_posts/2019-06-18-snowplow-snowflake-loader-0.5.0-released/index.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Snowplow Snowflake Loader 0.5.0 released
+title-short: Snowflake Loader 0.5.0
+tags: [snowflake, storage, relational databases]
+author: Enes
+category: Releases
+permalink: /blog/2019/06/18/snowplow-snowflake-loader-0.5.0-released/
+---
+
+We are pleased to announce version 0.5.0 of the [Snowplow Snowflake Loader][snowflake-loader-repo]! This release introduces bad row support with its [new format][new-bad-row-format-rfc] and brings performance improvement to the Snowflake Transformer with Kryo serialization.
+
+Read on below the fold for:
+
+1. [Bad row support](#bad-row)
+2. [Kryo serialization](#kryo-serialization)
+3. [Upgrading](#upgrading)
+4. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="bad-row">1. Bad Row Support</h2>
+
+Data quality has always been a huge focus for Snowplow. Being a non-lossy data pipeline is one of the central piece of this data quality puzzle. Indeed, other pieces such as data validity (through our schema validation technology) or data richness (through our enrichments) connect directly with the non-lossiness piece.
+
+In practice, for a Snowplow pipeline, non-lossiness meant that when something went wrong anywhere in the pipeline, instead of discarding it, the data impacted is parked as “bad” for later inspection, fixing and reprocessing. In Snowplow jargon, this bad data is called “bad rows”.
+
+With this release, we are adding bad row support to Snowflake Transformer with [its new format][new-bad-row-format-rfc]. From now on, Snowflake Transformer will continue to run when it encounters with unexpected data and it will write the bad row to separate place for further inspection instead of halting with exception.
+
+You have to specify where you want to store the bad rows in the config file. You can find detailed information about it in the [upgrading part](#upgrading).
+
+<h2 id="kryo-serialization">2. Kryo serialization</h2>
+
+With this release, we are starting to use [Kryo][kryo] library instead of Java serialization to serialize objects more quickly. This will bring performance improvement to Snowflake Transformer job.
```

Comment:
> instead of Java serialization

Technically, this is not entirely accurate. 

I'd rephrase this whole paragraph to be more like "Other changes" and mention that we've made improvements in serializiation which gave us X% performance boost.